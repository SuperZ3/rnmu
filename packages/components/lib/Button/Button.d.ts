export type ButtonProps = {
    a: string;
};
export declare const Button: () => import("react/jsx-runtime").JSX.Element;
