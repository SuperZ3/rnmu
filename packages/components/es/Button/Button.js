import { Text } from "react-native";
const Button = () => /* @__PURE__ */ React.createElement(Text, null, "Hello World!");
export {
  Button
};
