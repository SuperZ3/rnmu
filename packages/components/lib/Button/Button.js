"use strict";
Object.defineProperty(exports, Symbol.toStringTag, { value: "Module" });
const reactNative = require("react-native");
const Button = () => /* @__PURE__ */ React.createElement(reactNative.Text, null, "Hello World!");
exports.Button = Button;
