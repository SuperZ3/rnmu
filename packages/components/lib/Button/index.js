"use strict";
Object.defineProperty(exports, Symbol.toStringTag, { value: "Module" });
const Button_Button = require("./Button.js");
require("react-native");
exports.Button = Button_Button.Button;
