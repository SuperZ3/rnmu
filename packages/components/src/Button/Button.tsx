import { Text } from 'react-native';

export type ButtonProps = {
  a: string;
};

export const Button = () => <Text>Hello World!</Text>;
