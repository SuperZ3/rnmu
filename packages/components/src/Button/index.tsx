const Button = () => <>Hello World</>;

export default Button;
