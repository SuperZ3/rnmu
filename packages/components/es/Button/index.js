import { Button } from "./Button.js";
import "react-native";
export {
  Button
};
