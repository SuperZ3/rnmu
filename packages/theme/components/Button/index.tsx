import Button from '@rnmu/components/src/Button';

export default Button;
