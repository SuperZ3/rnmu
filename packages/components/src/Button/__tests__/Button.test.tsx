import { render } from '@testing-library/react-native';
import { Button } from '../Button';

describe('Button Component', () => {
  it('should match snapshot', () => {
    const button = render(<Button></Button>).toJSON();
    expect(button).toMatchSnapshot();
  });
});
