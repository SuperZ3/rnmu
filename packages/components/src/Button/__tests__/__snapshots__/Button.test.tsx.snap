// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component should match snapshot 1`] = `
<Text>
  Hello World!
</Text>
`;
